.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: go
+go:
+  - 1.9
```

Comment:
We only require 1.8 right?  This is probably fine but it's not strictly required.